aws/cf_template.go

```diff
@@ -151,7 +151,8 @@ func generateTemplate(spec *stackSpec) (string, error) {
 			Port:            cloudformation.Integer(80),
 			Protocol:        cloudformation.String(protocol),
 		})
-		if spec.denyInternalDomains {
+		// NLBs do not support Rules
+		if spec.denyInternalDomains && protocol != "TCP" {
```

Comment:
This should not be based on protocol but on the load balancer type, see how we check it elsewhere.

Reply:
good one, it actually makes even more sense to allow only ALBs instead of filtering out NLBs.